gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html

```diff
@@ -0,0 +1,38 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card class="app-card" appearance="outlined">
+  <mat-card-content class="app-card__content">
+    <div class="app-card__header">
+      <app-picture [picture]="application._links?.picture" [hashValue]="application.name" [size]="40" />
+      <div class="app-card__header__content">
+        <div class="m3-title-medium">{{ application.name }}</div>
+        <div class="m3-body-medium"><span i18n="@@owner">Owner: </span>{{ application.owner?.display_name }}</div>
+      </div>
+    </div>
+    <div class="app-card__description m3-body-medium">
+      @if (application.description) {
+        {{ application.description }}
+      } @else {
+        <span i18n="@@descriptionMissing">Description for this application is missing.</span>
+      }
+    </div>
+  </mat-card-content>
+  <mat-card-actions class="app-card__actions">
+    <button i18n="@@cardButton" mat-stroked-button color="primary" [routerLink]="['.', application.id]">Open</button>
```

Comment:
i18n id can be more specific to this component

Reply:
Done